test/log-iterator.spec.js

```diff
@@ -0,0 +1,344 @@
+'use strict'
+
+const assert = require('assert')
+const rmrf = require('rimraf')
+const Log = require('../src/log')
+const AccessController = Log.AccessController
+const IdentityProvider = require('orbit-db-identity-provider')
+const LogCreator = require('./utils/log-creator')
+
+// Test utils
+const {
+  config,
+  testAPIs,
+  startIpfs,
+  stopIpfs
+} = require('./utils')
+
+let ipfs, testIdentity, testIdentity2, testIdentity3, testIdentity4
+
+Object.keys(testAPIs).forEach((IPFS) => {
+  describe('Log - Iterator (' + IPFS + ')', function () {
+    this.timeout(config.timeout)
+
+    const testACL = new AccessController()
```

Comment:
`testACL` isn't really necessary anymore as it is the same as the Log default value

Reply:
Removed!